### PR TITLE
Add german translations for packaging taxonomies

### PR DESCRIPTION
**Pull request by jolesh**
**Description:**
Added German translations for packaging taxonomies.

### CODE REVIEW

File: taxonomies/packaging_materials.txt [hunk 1]
```diff
@@ -187,6 +191,7 @@ nl_be:Acrylonitril-butadieen-styreen, ABS
 <en:Plastic
 <en:Other plastics
 en:PLA - Polylactic acid, Polylactic acid
+de:PLA, Polylactide, PLA - Polylactide
```
Comment (stephanegigandet) [inline taxonomies/packaging_materials.txt:194]: by convention we currently put both the acronym and the full name as the main name

```suggestion
de:PLA - Polylactide, PLA, Polylactide
```

File: taxonomies/packaging_materials.txt [hunk 2]
```diff
@@ -187,6 +191,7 @@ nl_be:Acrylonitril-butadieen-styreen, ABS
 <en:Plastic
 <en:Other plastics
 en:PLA - Polylactic acid, Polylactic acid
```
Comment (stephanegigandet) [inline taxonomies/packaging_materials.txt:193]: not related to your change but we seem to be missing PLA

```suggestion
en:PLA - Polylactic acid, Polylactic acid, PLA
```

File: taxonomies/packaging_materials.txt (hunk 1 above)
Comment (jolo0815) [reply taxonomies/packaging_materials.txt]: Okay, thanks

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**